Flagsmith.FlagsmithClient/Extensions/DictionaryExtensions.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Flagsmith.Extensions
+{
+    internal static class DictionaryExtensions
```

Comment:
What's the purpose of this class? Is this some C# magic to override the `ForEach` method on a dictionary?

Reply:
Dictionary Collection in c# dont have Foreach method , that's why we created extension method on our own.